src/main/java/com/robothand/highqualitybot/Config.java

```diff
@@ -68,20 +70,26 @@ public static void loadConfig(String path) throws FileNotFoundException {
                     }
                     break;
 
+                case "logLevel":
+                    ch.qos.logback.classic.Logger root =
+                            (ch.qos.logback.classic.Logger)LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
```

Comment:
Why is ch.qos.logback.classic.Logger used here instead of org.slf4j.Logger?

Reply:
SLF4J's Logger does not have the `setLevel()` method, so we have to be a bit more specific here.